packages/modules/npm.py

```diff
@@ -1,21 +1,44 @@
 import requests
+import semver
 
 from packages.models import VersionedPackage
 
 NPM_REGISTRY_URL = "https://registry.npmjs.org"
 
 
-def get_versioned_package(name: str, version: str) -> VersionedPackage:
-    url = f"{NPM_REGISTRY_URL}/{name}/{version}"
+def get_package(name: str, range: str) -> VersionedPackage:
+    url = f"{NPM_REGISTRY_URL}/{name}"
 
-    response = requests.get(url)
-    npm_package = response.json()
+    npm_package = requests.get(url).json()
```

Comment:
There's no error-handling. That's probably what causes the crashing issues mentioned on the ticket.
Please handle errors (where the response might not be in JSON format).

```suggestion
    npm_package_response = requests.get(url)
    
    try:
        npm_package_response.raise_for_status()
        npm_package = npm_package_response.json()
    except Exception:  # Preferable update this with specific excppetions
        # Add logic here
```